Library/Formula/hbase.rb

```diff
@@ -29,6 +87,53 @@ def caveats; <<-EOS.undent
     EOS
   end
 
+  plist_options :manual => "#{HOMEBREW_PREFIX}/opt/hbase/bin/start-hbase.sh"
+
+  def plist; <<-EOS.undent
+    <?xml version="1.0" encoding="UTF-8"?>
+    <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+    <plist version="1.0">
+    <dict>
+      <key>KeepAlive</key>
+      <true/>
+      <key>Label</key>
+      <string>#{plist_name}</string>
+      <key>EnvironmentVariables</key>
+      <dict>
+       <key>HBASE_MASTER_OPTS</key><string> -XX:PermSize=128m -XX:MaxPermSize=128m</string>
+       <key>HBASE_LOG_DIR</key><string>#{var}/hbase</string>
+       <key>HBASE_HOME</key><string>#{opt_libexec}</string>
+       <key>HBASE_SECURITY_LOGGER</key><string>INFO,RFAS</string>
+       <key>HBASE_PID_DIR</key><string>#{rundir}</string>
+       <key>HBASE_NICENESS</key><string>0</string>
+       <key>HBASE_IDENT_STRING</key><string>root</string>
+       <key>HBASE_REGIONSERVER_OPTS</key><string> -XX:PermSize=128m -XX:MaxPermSize=128m</string>
+       <key>HBASE_OPTS</key><string>-XX:+UseConcMarkSweepGC</string>
+       <key>HBASE_ROOT_LOGGER</key><string>INFO,RFA</string>
+       <key>HBASE_LOG_PREFIX</key><string>hbase-root-master-#{`hostname`.strip}</string>
```

Comment:
The name of the logfile wouldn't change to the new hostname, but that's not a huge deal. Same thing happens if you don't restart hbase on a linux server whenever the hostname changes. And restarting hbase is generally considered a Bad Idea™.